docs/aca/06-aca-dapr-bindingsapi/index.md

```diff
@@ -449,39 +459,44 @@ az containerapp env dapr-component set `
 
 Next, we will add create the three Dapr bindings components using the component files created.
 
-!!! warning "Important"
+<!-- 10/23/2023 - Simon Kurtz - Commenting out as the current AZ CLI version does not error, but I am not yet ready to remove it (even though it is in source control history) -->
```

Comment:
If you didn't face this issue then the CLI is updated, the issue opened still not closed. We can remove this warning

Reply:
Affirmative that the CLI now addresses this correctly. I just removed the commented out warnings in the latest commit.